File: test/labview-test.js
var repl = require('nesh');
LV = require('../LabVIEW');
ctrls = {};
names = {};
data = [];

LVSock = require('dgram').createSocket('udp4');
LVSock.bind(10000);
LVSock.on('message', function(msg,err) {
    var ctrl = msg.toString().trim().split(' ')[1];
    LV.CtrlGetValue(ctrl,function(err,res) {
        console.log(names[ctrl],' ',res);
    });
});

LV.GetAllCtrls(__dirname + '\\Test.UDP.vi', function (err, res) {
    ctrls = res;
    for (var k in ctrls) names[ctrls[k]] = k;
});

repl.start('> ');

File: test/labview-test.lua
require('LabVIEW')

local dir = io.popen'cd':read'*l'
local ctrls = GetAllCtrls(dir..'\\Test.Queue.vi')
local names = {}
for k,v in pairs(ctrls) do names[v] = k end
while true do
local r = Queue('Test')
print(names[r[0]]..' '..r[1])
end